### PR TITLE
doc: update GSG and partitioned documentation

### PR DESCRIPTION
**Pull request by jackwhich**
1. Update launch script and replace the "apt-get" command with "apt" in GSG documentation.
2. Update the tag to v2.7 in partitioned documentation.

Signed-off-by: zhongzhenx.liu <zhongzhenx.liu@intel.com>

### CODE REVIEW

**Comment by acrnsi** [comment]
Can one of the admins verify this patch?

File: doc/getting-started/getting-started.rst [hunk 1]
```diff
@@ -528,18 +528,18 @@ Build ACRN
...
-         cp ~/acrn-work/acrn-hypervisor/build/hypervisor/serial.conf $disk/
          cp ~/acrn-work/my_board/output/launch_user_vm_id3.sh $disk/
          cp ~/acrn-work/acpica-unix-20210105/generate/unix/bin/iasl $disk/
          cp ~/acrn-work/acrn-hypervisor/build/acrn-2.7-unstable.tar.gz $disk/
          sync && sudo umount $disk/
 
       Even though our sample default scenario defines six User VMs, we're only
       going to launch one of them, so we'll only need the one launch script.
-
+
```
Comment (gvancuts) [inline doc/getting-started/getting-started.rst:538]: Looks like you have extra spaces here.
```suggestion

```

File: doc/getting-started/getting-started.rst [hunk 2]
```diff
@@ -783,13 +783,14 @@ Launch the User VM
    .. code-block:: bash
       :emphasize-lines: 4
 
-      acrn-dm -A -m $mem_size -s 0:0,hostbridge -U 615db82a-e189-4b4f-8dbb-d321343e4ab3 \
+      acrn-dm -A -m $mem_size -s 0:0,hostbridge \
```
Comment (gvancuts) [inline doc/getting-started/getting-started.rst:786]: Is this consistent with how the tool generates the launch script?

File: doc/getting-started/getting-started.rst [hunk 3]
```diff
@@ -528,18 +528,18 @@ Build ACRN
...
 
       Even though our sample default scenario defines six User VMs, we're only
       going to launch one of them, so we'll only need the one launch script.
-
+      
       .. note:: The :file:`serial.conf` is only generated if non-standard
          vUARTs (not COM1-COM4)
-         are configured for the Service VM in the scenario XML file.
+         are configured for the Service VM in the scenario XML file. 
+         Please copy this file :file:`cp ~/acrn-work/acrn-hypervisor/build/hypervisor/serial.conf $disk/`
```
Comment (dbkinder) [inline doc/getting-started/getting-started.rst:542]:
```suggestion
         Please copy the ``serial.conf`` file using::
         
            cp ~/acrn-work/acrn-hypervisor/build/hypervisor/serial.conf $disk/                 
```

**Review by dbkinder — CHANGES_REQUESTED** [review]
Looks good, with a couple of edits

**Review by dbkinder — APPROVED** [review]
other than @gvancuts question, LGTM.

**Comment by dbkinder** [comment]
I found a bug in the GSG that requires quoting the uses of "$disk" (it might contain a space).  That PR was recently merged and introduced a merge conflict with your PR.   A rebase should make things better...

File: doc/getting-started/getting-started.rst [hunk 4]
```diff
@@ -778,18 +780,35 @@ Launch the User VM
...
       :emphasize-lines: 4
 
-      acrn-dm -A -m $mem_size -s 0:0,hostbridge -U 615db82a-e189-4b4f-8dbb-d321343e4ab3 \
+      acrn-dm -A -m $mem_size -s 0:0,hostbridge \
          --mac_seed $mac_seed \
          $logger_setting \
-         -s 7,virtio-blk,/home/acrn/acrn-work/ubuntu-18.04.5-desktop-amd64.iso \
-         -s 8,virtio-net,tap_YaaG3 \
-         -s 6,virtio-console,@stdio:stdio_port \
+         -s 9,virtio-blk,./YaaG.img \
```
Comment (gvancuts) [inline doc/getting-started/getting-started.rst:791]: That's not consistent with the statement a few lines above, can you fix it?

File: doc/getting-started/getting-started.rst (hunk 2 above)
Comment (gvancuts) [reply doc/getting-started/getting-started.rst:804]: ?

**Review by gvancuts — CHANGES_REQUESTED** [review]

**Review by dbkinder — CHANGES_REQUESTED** [review]

**Review by dbkinder — APPROVED** [review]
I made my recommended changes directly to your branch.  LGTM now, thanks!

**Pull request merged**